src/snapshot.zig

```diff
@@ -273,6 +273,10 @@ const Error = error{
 };
 
 fn processSnapshotFile(gpa: Allocator, snapshot_path: []const u8, maybe_fuzz_corpus_path: ?[]const u8) !bool {
+
+    // Log the file path that was written to
+    log("processing snapshot file: {s}", .{snapshot_path});
```

Comment:
Switched to the start so using `--verbose` you can isolate which snapshot is causing a panic.